android/app/generate_strings.py

```diff
@@ -87,15 +87,17 @@ def main():
 
     src_strings = read_catalog(join(locale_dir, "messages.pot"))
     ids = make_ids(src_strings)
+
+    log(f"Writing to {args.res_dir}")
```

Comment:
Intentionally left in?

Reply:
Yes, it used to write to the standard strings.xml locations under src/res, but now it uses a custom location in the build directory, so I thought it would be a good idea to log that so the developer knows where to look.